jekyll/_cci2/migrate-from-launch-agent-to-machine-runner-3-on-linux.adoc

```diff
@@ -3,21 +3,19 @@ contentTags:
   platform:
   - Cloud
 ---
-= Migrate from launch agent to machine runner 3.0 - Open preview
+= Migrate from launch agent to machine runner 3.0 on Linux - Open preview
 :page-layout: classic-docs
 :page-liquid:
 :page-description: Steps to migrate from using launch agent to the machine runner 3.0 preview
 :icons: font
 :toc: macro
 :toc-title:
-:machine:
 :experimental:
```

Comment:
This is no longer "experimental".

Reply:
The "experimental" tag is on a bunch of our other non-experimental docs (like the OG runner install docs) so I think it's related to something specific to the doc engine and not necessarily the doc itself.

Reply:
Yeah this is on all pages to enable some Asciidoc formatting options :-D https://docs.asciidoctor.org/asciidoc/latest/syntax-quick-reference/#keyboard-button-and-menu-macros